src/fonduer/parser/spacy_parser.py

```diff
@@ -15,7 +17,7 @@
     raise Exception("spaCy not installed. Use `pip install spacy`.")
 
 
-class Spacy(object):
+class Spacy(LingualParser):
```

Comment:
Rename to `SpacyParser`?

Reply:
Renamed as suggested at b07538a706aa03a33ebda4510d2144309da7dc28.